@AH-64D Apache Longbow/Addons/fza_ah64_ase/functions/fn_rlwr.sqf

```diff
@@ -71,7 +71,6 @@ if (_rlwrPwr == "on") then {
     } foreach getSensorThreats _heli;
     _rlwrObjects = [_rlwrObjects, [], {_x # 0}, "DESCEND"] call BIS_fnc_sortBy;
 } else { _rlwrObjects = []; };
-_heli setVariable ["fza_ah64_ase_rlwrObjects", _rlwrObjects];
 _rlwrCount = count _rlwrObjects;
 _heli setVariable ["fza_ah64_ase_rlwrCount", _rlwrCount];
```

Comment:
Shouldn't this also be public according to defs in `eventInit`? If this one was missed there may be others. I would recommend doing a Ctrl+Shift+F of all the variables you are modifying in order to validate all usage sites

Reply:
left it intentionally, one less variable to make public, and each seat is able to access the funcs to calculate identical outputs